docs/SETTING_UP_A_NEW_INSTANCE.md

```diff
@@ -10,20 +10,38 @@ DCAF has a lot of the surrounding infrastructure set up in such a way that it's
 
 ### Create a new heroku app
 
-[Click this here link to spin up a new app](https://heroku.com/deploy?template=https://github.com/DCAFEngineering/dcaf_case_management). Follow the directions to fill out environment variables and configuration. (Ask our slack channel if you have questions.)
+[Click this here link to spin up a new app](https://heroku.com/deploy?template=https://github.com/DCAFEngineering/dcaf_case_management). By deploying using this link, the following accounts will be automatically provisioned and managed via Heroku (as heroku add-ons). This means that each individual app deployed with this link will have its own unique accounts with the following:
+
+* mLabs -- this is a mongodb data service
+* Sendgrid -- this is used as the email service
+* Logentries -- used for application logging
+* Scheduler -- used to run tasks on a cron schedule
+
+*Note -- if you choose to deploy this app manually rather than via the link provided above, you will need to handle your own set up for your mongodb, email, logging, and cron services.*
+
+Follow the directions to fill out necessary environment variables and configuration for your DARIA instance. (Ask our slack channel if you have questions.)
 
 * Go into logentries and add anyone who needs to be added to the alerts. This should be just people who need to know about application errors, pretty much.
-* Set up a google cloud account and generate OAuth credentials (Google Cloud Platform -> API Manager -> Credentials -> Create Credentials, can be set up alongside existing creds for different environments)
 * Set up an uptime monitor (we recommend StatusCake!)
 * Click on `Scheduler` and set up the scheduled cleanup job:
 - `$ rake nightly cleanup`, dyno size hobby, frequency daily, 08:00 UTC
-
-Optionally but recommended:
-
-## Optional steps
-
-* Set `SKYLIGHT_AUTH_TOKEN` (get it from Skylight, this is a performance monitoring service)
-* Set `SQREEN_TOKEN` (get it from Sqreen, this is a security service)
+* Set up Google OAuth credentials
+  * Set up a google account to administer this app (or choose an existing account you own)
+  * Once logged in, navigate to https://console.developers.google.com/apis/dashboard
+  * Add a project (note -- credentials can be set up alongside existing creds for different environments)
+  * Enable the Google+ API (this is necessary to use google for sign in to the app)
```

Comment:
I got hung up on this initially. The error messaging you receive from google when oauth doesn't work is also not particularly helpful in pushing toward this solution. Curious, do you all just have Google+ api enabled or do you also have Contacts and People APIs enabled? I found some conflicting sources on which apis were indeed necessary to have enabled for the creds to work for oauth sign-in and after testing manually found that Google+ was the minimum api to enable for login to work.

Reply:
yeah, you're right to call this out. Google is phasing out the google+ api (see https://developers.google.com/+/api-shutdown and #1560 ). Based on https://github.com/zquestz/omniauth-google-oauth2/pull/344 and some of the issues I think we can do without turning the G+ API on after we do a gem version bump. I'll fire up a PR for that. sorry you got burned by this!

Reply:
All good! I'll take out that step about G+ api since it won't be relevant once your PR merges.